File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int answer = 0;
        int servings = n / 10; 
        
        int yakitoriCost = n * 12000;
        int drinkCost = k * 2000;
        
        if (servings > 0) {
            drinkCost -= servings * 2000; 
        }

        answer = yakitoriCost + drinkCost;
        
        return answer;
    }
}

File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.py
def solution(n, k):
    answer = 0
    servings = n // 10; 
        
    yakitoriCost = n * 12000
    drinkCost = k * 2000
        
    if servings > 0:
        drinkCost -= servings * 2000

    answer = yakitoriCost + drinkCost
    return answer